x-pack/platform/packages/shared/kbn-evals/src/evaluators/correctness/index.ts

```diff
@@ -16,6 +16,29 @@ import {
   calculateProceduralFidelityScore,
 } from './scoring';
 import type { CorrectnessAnalysis } from './types';
+import { parseSelectedEvaluators } from '../filter';
+
+const QUALITATIVE_EVALUATOR_NAME = 'Correctness Analysis';
+const FACTUALITY_EVALUATOR_NAME = 'Factuality';
+const RELEVANCE_EVALUATOR_NAME = 'Relevance';
+const SEQUENCE_ACCURACY_EVALUATOR_NAME = 'Sequence Accuracy';
```

Comment:
Are we really fine with white spaces in selectable names?

```
export function parseSelectedEvaluators() {
  return (
    process.env.SELECTED_EVALUATORS?.split(',').map((selectedEvaluator) =>
      selectedEvaluator.trim()
    ) ?? []
  );
}
```

As soon as I see a comma based split, and the things being used via CLI params, I can't avoid thinking that white space shouldn't be permitted (even if technically it works)